csrc/serde/fusion_cache.fbs

```diff
@@ -387,6 +387,7 @@ table SegmentedGroup {
 
 // The SegmentedFusion represents a segmented fusion graph that owns the segmented groups.
 table SegmentedFusion {
+  valid: bool;
```

Comment:
nitpick: can we actually put a comment here explaining why we have the `valid` field. I imagine it would be very confusing to folks going through this.
Maybe a link the issue 1706 would suffice.